relay-server/src/actors/processor.rs

```diff
@@ -2276,6 +2276,8 @@ mod tests {
     }
 
     #[test]
+    #[ignore = "The current Register panics if the Addr of an Actor (that is not yet started) is
```

Comment:
Leaving a comment here to make this visible. These tests must be fixed in a follow up when `Addr`s allow to mock services.